### DISCUSSION TITLE
What is references-data.bib ?

### DISCUSSION THREAD

**Discussion by kilianhauber** [timlrx/tailwind-nextjs-starter-blog#421] [Q&A] [answered]
# What is references-data.bib ?
Hello, I was wondering what the "references-data.bib" file is, because I can't do much with what is written in it. Is this file important at all or can I delete it? 
- Kilian Hauber
![unknown](https://user-images.githubusercontent.com/44700745/159038169-63d9e259-b5ed-4cd1-a039-18bf33be6ad5.png)

**Comment by timlrx** [comment, marked as answer]
Hi @kilianhauber. It's a sample bibliography file, if you want to do academic style citations. Feel free to delete it if you are not using that feature.
